.github/buildomat/jobs/image.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#:
+#: name = "image"
+#: variety = "basic"
+#: target = "helios"
+#: rust_toolchain = "nightly"
+#: output_rules = [
+#:   "/out/*",
+#: ]
+#:
+#: [[publish]]
+#: series = "image"
+#: name = "propolis-server.tar.gz"
+#: from_output = "/out/propolis-server.tar.gz"
+#:
+#: [[publish]]
+#: series = "image"
+#: name = "propolis-server.sha256.txt"
+#: from_output = "/out/propolis-server.sha256.txt"
+#:
+
+set -o errexit
+set -o pipefail
+set -o xtrace
+
+cargo --version
+rustc --version
+
+banner build
+ptime -m cargo build --release --verbose
```

Comment:
Should this build only the `propolis-server` package required for the packaging?

Reply:
Sure thing, done.